reference/carts.sf.yml

```diff
@@ -250,6 +250,45 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/deleteCartsItems'
+  '/carts/{cartId}/currency':
+    post:
+      tags:
+        - Cart Currency
+      summary: Update Cart currency
+      description: |-
+        Update currency of the *Cart*. 
+        Promotions and gift certificates that don't apply to the new currency will be removed from your cart.
+        You cannot update the cart currency if the draft order cart or the cart contains a manual discount.
+
+        <!-- theme: info -->
+        > #### Note
+        > The Send a Test Request feature is not currently supported for this endpoint.
+      operationId: addCartCurrency
+      parameters:
+        - name: cartId
+          in: path
+          description: This cart's unique ID.
+          required: true
+          schema:
+            type: string
+            format: UUID
+      responses:
```

Comment:
@traci707 any headway on documenting error response codes?

Reply:
@slsriehl thank you for reminding me.